ui/components/app/transaction-detail-item/transaction-detail-item.component.js

```diff
@@ -22,14 +22,14 @@ export default function TransactionDetailItem({
         <Typography
           color={detailTitleColor}
           fontWeight={FONT_WEIGHT.BOLD}
-          variant={TYPOGRAPHY.H6}
+          variant={TYPOGRAPHY.H7}
```

Comment:
I've changed this because according to our design we should use H7. Benefit is that now text fits on one line: 
![image](https://user-images.githubusercontent.com/80175477/127200553-7b03df6f-6b6f-4983-a516-2a2bec4b6b1b.png)